src/main/java/com/comphenix/protocol/utility/MinecraftVersion.java

```diff
@@ -36,6 +36,11 @@
  */
 public final class MinecraftVersion implements Comparable<MinecraftVersion>, Serializable {
 
+	/**
+	 * Version 1.19.3 - introducing feature preview
+	 * TODO: is this a good idea?

```

Comment:
it's probably fine since it's a snapshot API which is obviously subject to change